lib/helpers/response-wrapper.js

```diff
@@ -0,0 +1,9 @@
+class responseWrapper {
+  static respond({ res, status }) {
```

Comment:
this could just be this 
```
 static respond({ res, status }) {
      return res.status(status).send({ data })
  }
```

where is data coming from btw? Your method parameters are just `res` and `status` but you returning data.